modules/ocf/manifests/packages/gnupg.pp

```diff
@@ -0,0 +1,5 @@
+class ocf::packages::gnupg {
+  if $::lsbdistcodename != 'jessie' {
+    package { 'dirmngr':; }
+  }
+}
```

Comment:
Agreed, a comment here (and perhaps in `site.pp`) would help to make it clearer why we need this package specifically.